src/thingset_bin.c

```diff
@@ -703,6 +750,42 @@ inline void thingset_bin_setup(struct thingset_context *ts, size_t rsp_buf_offse
                            ts->rsp_size - rsp_buf_offset, 1);
 }
 
+#ifdef CONFIG_THINGSET_PROGRESSIVE_IMPORT_EXPORT
```

Comment:
Doesn't this belong to the other PR?

Reply:
Yes, sorry, I've got 3 things on the go as you can see and I made a bit of a mess. Think it's OK now.